services/brig/deb/opt/brig/templates/de/team/email/invitation.txt

```diff
@@ -3,15 +3,21 @@
 ${brand_label_url} [${brand_url}]
 
 EINLADUNG ZUM TEAM
-${inviter} hat Sie auf ${brand} zu einem Team eingeladen. Klicken Sie bitte auf
-den unteren Button, um die Einladung anzunehmen.
+${inviter} hat Sie eingeladen, dem Team ${team_name} auf ${brand} beizutreten.
```

Comment:
We removed `team_name` from the messages at some point in the past: it's prone to abuse, as the team name can be anything, including a link to a dubious website or offensive language.

See also https://github.com/wireapp/wire-server/pull/136 and the item `Removal of user generated text input from team invitations`